src/Raven.Server/Commercial/SetupInfo.cs

```diff
@@ -1,79 +1,148 @@
 ﻿using System;
 using System.Collections.Concurrent;
 using System.Collections.Generic;
+using System.IO;
 using System.Linq;
 using System.Security.Cryptography.X509Certificates;
+using System.Threading.Tasks;
+using Raven.Client;
 using Raven.Client.Documents.Operations;
 using Raven.Client.Documents.Operations.Configuration;
 using Raven.Client.Util;
+using Raven.Server.Commercial.SetupWizard;
 using Sparrow.Json.Parsing;
 using Sparrow.Logging;
 
 namespace Raven.Server.Commercial
 {
-    public class SetupInfo
+    public class NodeInfo
+    {
+        public string PublicServerUrl { get; set; }
+        public string PublicTcpServerUrl { get; set; }
```

Comment:
@gregolsky what is this property `PublicTcpServerUrl` in the Studio ?
I see it is part of the `NodeInfo` class
but I also see that the Studio does not fill in any data for it

Reply:
So which UI element represent it in the image ?

![image](https://user-images.githubusercontent.com/17376974/182859254-67a8231f-7eb0-48f0-b2ee-7b17f76ff417.png)

Reply:
currently, the Studio does not set this property at all

Reply:
@gregolsky when I look at a zip file that has been generated
I see that the server has placed the following inside the settings.json
`  "PublicServerUrl.Tcp": "tcp://a.12345.development.run:38888"`

So I guess it is ok for the studio to not pass any value for it - as is now - correct ?